src/app/content/components/BookBanner.tsx

```diff
@@ -141,7 +141,7 @@ export const BarWrapper = styled.div<BarWrapperProps>`
   ${disablePrint}
 
   top: 0;
-  padding: 0 ${theme.padding.page.desktop}rem;
+  padding: 0 3.2rem;
```

Comment:
Isn't this the same thing?

Reply:
At some point in my work, I changed this `theme.padding.page.desktop`, hence the discrepancy. Good catch :)

Reply:
I figured!